models/camelot/arbitrum/camelot_arbitrum_schema.yml

```diff
@@ -176,3 +176,46 @@ models:
       - *tx_from
       - *tx_to
       - *evt_index
+
+  - name: camelot_arbitrum_pair_fee_rates
+    meta:
+      blockchain: arbitrum
+      sector: dex
+      project: camelot
+      contributors: whale_hunter
+    config:
+      tags: ["arbitrum", "camelot", "whale_hunter"]
+    description: >
+      Camelot fee rates over time across all pairs and versions on Arbitrum.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - minute
+            - blockchain
+            - pair
+            - version
+    columns:
+      - blockchain
```

Comment:
i think you need `- &blockchain` format here. leaving out `&` make dbt angry.
however, you also use this above in this schema file.
you should either use:
- `- *blockchain`
- drop this line and go right into `name: blockchain`